middlewares/team.middleware.js

```diff
@@ -361,28 +360,6 @@ async function updateHackerTeam(req, res, next) {
     return next();
 }
 
-/**
- * @async
- * @function findById
- * @param {{body: {id: ObjectId}}} req 
- * @param {*} res 
- * @return {JSON} Success or error status
- * @description Finds a team by it's mongoId that's specified in req.param.id in route parameters. The id is moved to req.body.id from req.params.id by validation.
- */
-async function findById(req, res, next) {
```

Comment:
Is this being removed because it's being currently unused?

Reply:
There is another function definition in this file called `findById`, lol